tests/ott/swift/Example/Tests/TestConfig.template.plist

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>serverURL</key>
+	<string>@URL@</string>
```

Comment:
Hi @srivkas,
I think the place holder for the service URL/endpoint should be "@SERVICE_URL@", not "@URL@". Frankly, "@API_ENDPOINT@" would have been much better as its the industry term but since all our other clients use "@SERVICE_URL@" we might as well stick to that:)